vars/isRelease.groovy

```diff
@@ -0,0 +1,30 @@
+def call(Map args) {
+
+    def isRelease = isReleaseBranch()
+
+    if (env.BRANCH_NAME == 'master') {
```

Comment:
it is posible to use .equals(Object) ?
env.BRANCH_NAME.equals('master')
Also, 
can we make sure that env.BRANCH_NAME is not null.

Reply:
["Groovy overloads the == operator and maps it to the equals() method."](https://mrhaki.blogspot.com/2009/09/groovy-goodness-check-for-object.html)

Reply:
I think that suggestion will work. I will make the change. wrt `null`, the context that this code is run under `env.BRANCH_NAME` should never be null, so we take it as a given. Good question though.

Reply:
Changed as suggested.

Reply:
Are we suggesting checking `env` for `null` or `env.BRANCH_NAME`?  if `env.BRANCH_NAME` is null then `env.BRANCH_NAME == 'master'` will evaluate to `false` as intended.

Reply:
reverted change made.